### PR TITLE
`ashok2022re`

### PR DESCRIPTION
**Pull request by ashok-arjun**
## Project Name

`ashok2022re`

*Note*: The initial `citekey` was `ashok2022when` and it has been requested to be changed to `ashok2022re` in PR #76. 

## Compiled PDF

[Compiled Pdf](2021/ashok2022re/journal/article.pdf)

## Metadata Checklist

Ensure that you have added the following metadata information (put an x to the checkbox items) in `<paper_citekey>/journal/metadata.yaml`.

- [x] `title` field exists and it starts with `[Re] `.
- [x] `authors` contain author information with affiliations.
- [x] [ORCID id](https://orcid.org/) is provided for authors.
- [x] One contact author is added using `*`.
- [x] Github `url` is provided in `code` section.
- [x] [SWH identifier](https://archive.softwareheritage.org/save/) is provided in `swh`.
- [x] `replication` section is populated with the necessary information about the paper you reproduced.
- [x] `abstract` field is populated with your paper abstract (including double quotes)
- [x] `domain` should contain "ML Reproducibility Challenge 2021"
- [x] `review` section contains the url to the OpenReview forum of your paper.
- [ x Fields `contributors`, `dates`, `article` and `journal` are kept blank, as those will be populated by the Area Chairs / ReScience editors.

## Build Errors

1. The article fails to link any citation or reference. I get these errors:

```bash
  Citation 'aircrafts' on page 5 undefined on input line 86                                                                                         
  Citation 'cars' on page 5 undefined on input line 86                                                                                              
  Citation 'chen2018a' on page 4 undefined on input line 73                                                                                         
  Citation 'chen2018a' on page 4 undefined on input line 81                                                                                         
  Citation 'chen2018a' on page 6 undefined on input line 128                                                                                        
  Citation 'chen2018a' on page 7 undefined on input line 139                                                                                        
  Citation 'cub' on page 5 undefined on input line 86                                                                                               
  Citation 'dogs' on page 5 undefined on input line 86                                                                                              
  Citation 'finn2017model' on page 4 undefined on input line 81                                                                                     
  Citation 'flowers' on page 5 undefined on input line 86                                                                                           
  Citation 'guo2020broader' on page 9 undefined on input line 360
  Citation 'inat' on page 5 undefined on input line 92
  Citation 'mini' on page 5 undefined on input line 86
  Citation 'noroozi2016unsupervised' on page 4 undefined on input line 61
  Citation 'openimages' on page 5 undefined on input line 92                                                                                        
  Citation 'protonets' on page 3 undefined on input line 55                                                                                         
  Citation 'protonets' on page 4 undefined on input line 81                                                                                         
  Citation 'rengithub' on page 5 undefined on input line 90                                                                                         
  Citation 'sung2018learning' on page 4 undefined on input line 81                                                                                  
  Citation 'yao2github' on page 5 undefined on input line 90                                                                                        
  Reference `fig:20-40-60-80' on page 7 undefined on input line 186                                                                                 
  Reference `fig:20-60-other' on page 8 undefined on input line 259 
  Reference `fig:Conv4' on page 9 undefined on input line 286
  Reference `fig:Res-18-small' on page 7 undefined on input line 139
  Reference `fig:domain' on page 8 undefined on input line 274
  Reference `fig:lowres' on page 7 undefined on input line 186
  Reference `sec:claims' on page 9 undefined on input line 400
  Reference `ss:comp_req' on page 12 undefined on input line 7
  Reference `table:20_ssl' on page 7 undefined on input line 186
  Reference `table:20_ssl_others' on page 8 undefined on input line 259
  Reference `table:cdfsl_conv' on page 9 undefined on input line 360
  Reference `table:cdfsl_resnet' on page 9 undefined on input line 360
  Reference `table:conv_arch' on page 4 undefined on input line 81
  Reference `table:conv_few_shot' on page 9 undefined on input line 286
  Reference `table:conv_jigsaw' on page 9 undefined on input line 288
  Reference `table:conv_rotation' on page 9 undefined on input line 288
  Reference `table:domain' on page 8 undefined on input line 274
  Reference `table:epoch_time' on page 6 undefined on input line 116
  Reference `table:grey_low_res' on page 7 undefined on input line 186
  Reference `table:mini_imagenet' on page 7 undefined on input line 139
  Reference `table:mini_imagenet_both' on page 9 undefined on input line 286
  Reference `table:resnet_arch' on page 4 undefined on input line 81
  Reference `table:resnet_few_shot' on page 7 undefined on input line 139
  Reference `table:seed_conv' on page 9 undefined on input line 286
```

2. Not a build error, but the figures and tables generated in `article.pdf` are terribly misaligned. I'm not sure why, because this does not happen in the OpenReview pdf.

### CODE REVIEW

File: 2021/ashok2022re/openreview/packages.tex [hunk 1]
```diff
@@ -0,0 +1,34 @@
+% Import packages here
+\setcounter{secnumdepth}{3} % default value for 'report' class is "2"
+
+% \usepackage[preprint]{neurips_2019}
+
+\usepackage[utf8]{inputenc} % allow utf-8 input
+\usepackage[T1]{fontenc}    % use 8-bit T1 fonts
```
Comment (koustuvsinha) [inline 2021/ashok2022re/openreview/packages.tex:7]: Please remove this import (stated in README)

File: 2021/ashok2022re/openreview/packages.tex [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+\usepackage{amsfonts}       % blackboard math symbols
+\usepackage{nicefrac}       % compact symbols for 1/2, etc.
+\usepackage{microtype}      % microtypography
+\usepackage{makecell}
+\usepackage{textcomp}
+\usepackage{graphicx}
+\usepackage{multirow}
+\usepackage{float}
+
+\usepackage[dvipsnames]{xcolor}
```
Comment (koustuvsinha) [inline 2021/ashok2022re/openreview/packages.tex:20]: Please remove this import (stated in README)

File: 2021/ashok2022re/openreview/packages.tex (hunk 1 above)
Comment (ashok-arjun) [reply 2021/ashok2022re/openreview/packages.tex]: Done!

File: 2021/ashok2022re/openreview/packages.tex (hunk 2 above)
Comment (ashok-arjun) [reply 2021/ashok2022re/openreview/packages.tex]: Done!

File: 2021/ashok2022re/openreview/packages.tex [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+\usepackage[normalem]{ulem}
+\newif{\ifhidecomments}
+
+\usepackage{graphicx}
+\usepackage{capt-of}% or \usepackage{caption}
+\usepackage{booktabs}
+\usepackage{varwidth}
+\usepackage{subfig}
+\usepackage{caption,stackengine}
+
```
Comment (koustuvsinha) [inline 2021/ashok2022re/openreview/packages.tex:23]: Please add the package `multirow` as you are using it in your tables:

```
\usepackage{multirow}
```

Comment (ashok-arjun) [reply 2021/ashok2022re/openreview/packages.tex:24]: Done! @koustuvsinha

File: 2021/ashok2022re/journal/metadata.yaml [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+
+
+# Code URL and DOI/SWH (url is mandatory for replication, doi after acceptance)
+# You can get a DOI for your code from Zenodo, or an SWH identifier from
+# Software Heritage.
+#   see https://guides.github.com/activities/citable-code/
+code:
+  - url: https://github.com/ashok-arjun/MLRC-2021-Few-Shot-Learning-And-Self-Supervision/
+  - doi: 10.5281/zenodo.6508499
+  - swh: swh:1:dir:aaaaaaaaaaaaaaaaaaaaaaaaaaaaa
```
Comment (koustuvsinha) [inline 2021/ashok2022re/journal/metadata.yaml:47]: Hi, please provide the correct SWH id ASAP.

Comment (ashok-arjun) [reply 2021/ashok2022re/journal/metadata.yaml]: Done!

**Pull request merged**